pkgs/development/compilers/llvm/16/libcxx/default.nix

```
@@ -62,7 +60,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ninja python3 ]
     ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
 
-  buildInputs = lib.optionals (!headersOnly) [ cxxabi ];
+  buildInputs = (
+    lib.optional (!headersOnly) cxxabi ++
+    lib.optional ((stdenv.hostPlatform.useLLVM or false) && !stdenv.isDarwin) libunwind
```

Darwin isn’t `useLLVM`, so it doesn’t need to be included in the conditional.

```suggestion
    lib.optional (stdenv.hostPlatform.useLLVM or false) libunwind
```